src/app/modules/main/chat_section/module.nim

```diff
@@ -200,6 +219,7 @@ method load*(self: Module, events: EventEmitter,
     self.buildCommunityUI(events, settingsService, contactService, chatService, communityService, messageService)
   else:
     self.buildChatUI(events, settingsService, contactService, chatService, communityService, messageService)
+    self.initContactRequestsModel() # we do this only in case of chat section (not in case of communities)
```

Comment:
@jrainville I think no, since you're able to access and accept/reject contact request only from the `Chat` section, there is no other place in the app where you can do that from.

Reply:
That's true as far as the designs go. I wonder if it might change someday, since it's not purely a chat feature. Anyway, we can move it if we ever need to show the contact requests in the profile as well.

Reply:
When we have a new design(request) we will move it to appropriate place then, till then its place is here as following the architecture we have modules should and UI parts should be 1:1 mapped. So once it's moved we will maybe introduce a new module, but services are ready and supports all necessary actions.